### PR TITLE
Poncho Colour Variants, Clown Box Cost Reduction

### PR DESCRIPTION
**Pull request by assortedbeads**
## About The Pull Request
This PR

- Adds six more colour variants for the brown poncho to the loadout under "poncho selection"
- Adds recipes to craft the the new colour variants in the crafting menu
- Updates icons to the new poncho shape
- Adds a clown poncho (this was done under duress)
Purple Ponchi Red Ponchi White Ponchi Black Ponchi Blue Ponchi Green Ponchi Longer Poncho Front

Clown Poncho
Artist Poncho Clown Poncho Clowncho Naked

## Why It's Good For The Game
Feedback was positive on the poncho being an accessory, and people brought up having more colours for varieties sake.
I've used already existing colour palettes for the variants.

## Changelog
:cl:
add: Added 5 new colours for poncho
add: Adds a clown poncho
add: Adds relevant recipes for crafting
add: Adds loadout options for ponchos
imageadd: added new icons for the new ponchos
tweak: The clown costume now costs 1 loadout point instead of 3
/:cl:

### CODE REVIEW

**Comment by Firefox13** [comment]
изображение
Isn't the outline is a bit too dark on this one?

**Comment by assortedbeads** [comment]
> изображение
> Isn't the outline is a bit too dark on this one?

Maybe? It's got the same outline as the black suit uniform though, since I used that ones palette

**Comment by assortedbeads** [comment]
> изображение Isn't the outline is a bit too dark on this one?

Black Ponchi
Made it lighter

**Review by Firefox13 — APPROVED** [review]
Yeah that's better

File: code/modules/client/preference_setup/loadout/lists/misc.dm [hunk 1]
```diff
@@ -18,7 +18,6 @@
 /datum/gear/clown
 	display_name = "clown pack"
 	path = /obj/item/storage/box/clown
-	cost = 3
```
Comment (Humonitarian) [inline code/modules/client/preference_setup/loadout/lists/misc.dm:21]: Doesn't this pack give multiple powerful items such as a banana peel?

Comment (Alliostra) [reply code/modules/client/preference_setup/loadout/lists/misc.dm:21]: oh. my. god. clown meta incoming??? literally unplayable

Comment (assortedbeads) [reply code/modules/client/preference_setup/loadout/lists/misc.dm:21]: Iirc it does but I dont think that's super strong? 
I don't play clown but I've never had an issue finding a slip item, and 3 points for a funny gimmick seems too much.

**Pull request merged**